framework/src/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -3106,7 +3106,7 @@ public boolean validateTypeOf(Tree tree) {
             }
             break;
         default:
-            type = atypeFactory.getAnnotatedType(tree);
+        	type = atypeFactory.getDefaultedAnnotatedType(tree);
```

Comment:
I agree on the whitespace change, it should be undone.

Suzanne and I discussed the motivation for this change here:

https://groups.google.com/d/msg/checker-framework-dev/L321YcG3srY/6wwnC8XKBgAJ

I don't think any of the standard type systems can be used to reproduce this. Once the updated Universe type checker implementation is finished, we'll have a test.